README.md

```diff
@@ -90,6 +90,38 @@ ros2 launch l3xz control.py
 **Note**: MikroTik [RBMetal5SHPn](https://mikrotik.com/product/RBMetal5SHPn).
 
 #### Cyphal Configuration
+##### How-to-config via `yakut`
+Configure `can0`:
+```bash
+sudo ./setup_slcan.sh --remove-all --basename can --speed-code 5 /dev/serial/by-id/usb-Zubax_Robotics_Zubax_Babel_*-if00
+```
+Configure `yakut`:
+```bash
+python3 -m pip install yakut
+yakut compile https://github.com/OpenCyphal/public_regulated_data_types/archive/refs/heads/master.zip
+. setup_yakut.sh
+```
+Dump all write-able registers into l3xz-cyphal-config.yaml
+```bash
+y rl 11,12,13,14,15,16,20,30,40,50 | y --yaml rb --only=mp > config/cyphal-config.yaml
+```
+Format the YAML file for better human readability via [yamlfmt](https://github.com/google/yamlfmt):
```

Comment:
This makes me suspect that we could use `--yaml-block` or `--yaml-alex` or some other option that emits YAML according to your preferences.

Reply:
Done, see here https://github.com/OpenCyphal/yakut/issues/83 :bow: